Client/WebFilters/AdBlockEngineManager.swift

```diff
@@ -185,20 +185,17 @@ public actor AdBlockEngineManager: Sendable {
             resultString = "not compiled"
```

Comment:
This file is just an output cleanup for debugging purposes. No real change here otherwise